stdlib/2/dircache.pyi

```diff
@@ -0,0 +1,10 @@
+# Source: https://hg.python.org/cpython/file/2.7/Lib/dircache.py
+
+from typing import List, Text
+
+def reset() -> None: ...
+def listdir(path: Text) -> List[Text]: ...
+
+opendir = listdir
+
+def annotate(head: Text, list: List[Text]) -> None: ...
```

Comment:
Because of invariance, I don't believe this will now allow a list of strs for the `list` argument. I think strictly speaking it should be a `Union[List[str], List[Text], List[Union[str, Text]]]`.